assignments/objects.js

```diff
@@ -18,27 +18,73 @@ const example = {
 }
 
 // Write your intern objects here:
+const Mitzi = {
```

Comment:
It is conventional to use `camelCase` for variable names. Why does it matter? Conventions allow for better communication between developers. If you see a camel-cased variable you know that the developer probably intended it to be the name of a function or a variable. `PascalCase` is used for naming special functions called *constructor functions*. You will learn more about this soon.